frontend/app/src/services/history/types.ts

```diff
@@ -50,42 +43,38 @@ export interface AssetMovement {
   readonly link: string;
 }
 
-export const EthTransaction = z.object({
-  txHash: z.string(),
-  timestamp: z.number(),
-  blockNumber: z.number(),
-  fromAddress: z.string(),
-  toAddress: z.string(),
-  value: NumericString,
-  gas: NumericString,
-  gasPrice: NumericString,
-  gasUsed: NumericString,
-  inputData: z.string(),
-  nonce: z.number()
-});
-
-export type EthTransaction = z.infer<typeof EthTransaction>;
-
-const EthTransactionWithMeta = z.object({
-  entry: EthTransaction,
-  ignoredInAccounting: z.boolean()
-});
-
-export type EthTransactionWithMeta = z.infer<typeof EthTransactionWithMeta>;
-
-export const Transactions = z.object({
-  entries: z.array(EthTransactionWithMeta),
-  entriesFound: z.number(),
-  entriesLimit: z.number(),
-  entriesTotal: z.number()
-});
-
-export type Transactions = z.infer<typeof Transactions>;
```

Comment:
Did this schema move elsewhere?

Reply:
Alright, I removed it for the sake of simplicity, now I know it and will adjust it again